scst-scan/ivs-prisma.hbs.md

```diff
@@ -66,9 +100,13 @@ spec:
 
 Where:
 
-- `USERNAME` is the access Key from the Prisma Cloud account.
-- `PASSWORD` is the secret Key from the Prisma Cloud account.
-- `ADDRESS` is the URL for Console from the Prisma Cloud account.
-- `PRISMA-SCANNER-IMAGE` is the Prisma scanner image with twistcli and podman.
+- `TARGET-IMAGE` is the image to be scanned. You must specify the digest.
+- `PRISMA-SCANNER-IMAGE` is the image containing the Prisma Cloud [twistcli](https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/tools/twistcli), [podman](https://podman.io/docs/installation), and utility to convert the Prisma summary report (JSON format) to a CycloneDX SBOM in XML format. The scan report summary produced by Prisma cannot be used as is and must be converted.
```

Comment:
```suggestion
- `PRISMA-SCANNER-IMAGE` is the image containing the Prisma Cloud [twistcli](https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/tools/twistcli), [podman](https://podman.io/docs/installation), and a utility to convert the Prisma summary report (JSON format) to a CycloneDX SBOM in XML format. The prisma scanner produced a proprietary output instead of community standard CycloneDX or SPDX. Because of this, the scan report summary produced by Prisma cannot be used as is and must be converted.  Please contact your account team if you wish to use Prisma and need this utility.
```

Reply:
Try to head off any questions on this.  Also may give us a bit more exposure to who is using it for now.